packages/frontend/editor-ui/src/stores/folders.store.ts

```diff
@@ -108,6 +113,49 @@ export const useFoldersStore = defineStore(STORES.FOLDERS, () => {
 		return await workflowsApi.getProjectFolders(rootStore.restApiContext, projectId);
 	}
 
+	async function fetchFoldersAvailableForMove(
+		projectId: string,
+		folderId?: string,
+		filter?: {
+			name?: string;
+		},
+	): Promise<FolderListItem[]> {
+		const folders = await workflowsApi.getProjectFolders(
+			rootStore.restApiContext,
+			projectId,
+			{
+				take: !filter?.name ? 5 : undefined,
```

Comment:
Magic number, would be nice if we set this as a variable on top and give it a clear name.

Reply:
This is also not needed anymore since we are not fetching the initial page